server_v2/lib/resumes/main.py

```diff
@@ -0,0 +1,54 @@
+from lib.OpenAI_Lib.main import *
+from helpers import *
+
+class Analyser:
+    def __init__(self, **kwargs):
+        self.model = kwargs.get('model_name')
+        pass
+
+    def evaluate_characterstics(jinja_templated_resume_text=""):
```

Comment:
add method parameter to the function, method = 'cli' or 'json' ,
if method is cli run the input logic, elif json take these inputs from dict